retail/2.7/modules/ROOT/pages/sfdc-customization-guide.adoc

```diff
@@ -2,30 +2,429 @@
+1. Enable the `Open Log` option and click the **Execute** button.
+
+Once the script completes, select the `Debug Only` filter option to show just the results of each create request. Verify that all fields have been successfully created.
+
+== Custom Fields
+
+The following fields must be created in the corresponding objects.
 
 [%header%autowidth.spread]
 |===
-| Field Name | Type | Updated Values
-| Type | Picklist | ADD_ON, CANCELLATION, INITIAL, JOURNAL, RENEWAL, RETURN, SUBSCRIPTION, UPGRADE
-| Status | Picklist | CANCELLED, CREATED, CONFIRMED, DELIVERED, IN_CART, IN_TRANSIT, INVOICED, LOST, PARTIALLY_SHIPPED, PICKUP_AVAILABLE, PROCESSING, REJECTED, RETURNED
+| Salesforce Object | Field Label | Field Name | Data Type | Length | Unique* | ExternalId
+
+| Account
+| Global Billing Address ID
+| Global_BillingAddress_Id
+| Text
+| 36
+| Y
+| Y
+
+| Account
+| Global Customer ID
+| Global_Customer_Id
+| Text
+| 36
+| Y
+| Y
+
+| Account
+| Global Email Address ID
+| Global_EmailAddress_Id
+| Text
+| 36
+| Y
+| Y
+
+| Account
+| Global Fax Number ID
+| Global_FaxNumber_Id
+| Text
+| 36
+| Y
+| Y
+
+| Account
+| Global Mailing Address ID
+| Global_MailingAddress_Id
+| Text
+| 36
+| Y
+| Y
+
+| Account
+| Global Mobile Number ID
+| Global_MobileNumber_Id
+| Text
+| 36
+| Y
+| Y
+
+| Account
+| Global Party ID
+| Global_Party_Id
+| Text
+| 36
+| Y
+| Y
+
+| Account
+| Global Phone Number ID
+| Global_PhoneNumber_Id
+| Text
+| 36
+| Y
+| Y
+
+| Account
+| Global Shipping Address ID
+| Global_ShippingAddress_Id
+| Text
+| 36
+| Y
+| Y
+
+| Account
+| Customer Status
+| Customer_Status
+| Picklist
+| -
+| N
+| N
+
+| Managed_External_Id__c
+| External Id
+| External_Id
+| Text
+| 80
+| N
+| N
+
+| Managed_External_Id__c
+| External Id Type
+| External_Id_Type
+| Picklist
+| -
+| N
+| N
+
+| Managed_External_Id__c
+| Object Id
+| Object_Id
+| Text
+| 80
+| N
+| N
+
+| Managed_External_Id__c
+| Object Type
+| Object_Type
+| Picklist
+| -
+| N
+| N
+
+| Managed_External_Id__c
+| Status
+| Status
+| Picklist
+| -
+| N
+| N
+
+| Managed_External_Id__c
+| Status Last Changed
+| Status_Last_Changed
+| Date/Time
+| -
+| N
+| N
+
+| Contact
+| Global_Individual_Id
+| Global_Individual_Id
+| Text
+| 36
+| Y
+| Y
+
+| Order
+| Global_SalesOrder_Id
+| Global_SalesOrder_Id
+| Text
+| 36
+| Y
+| Y
+
+| Product2
+| Global_Product_Id
+| Global_Product_Id
+| Text
+| 36
+| Y
+| Y
+
 |===
 
+*Unique fields should be case-sensitive
+
+=== Create via Apex Script
+
+An Apex class called `CustomFieldUtility` can be used to create custom fields programmatically. The source for this class can be found in the https://anypoint.mulesoft.com/exchange/org.mule.examples/accelerator-common-resources-src[Accelerator Common Resources^] project. Download this project and follow the steps below.
+
+. Click on Salesforce Setup icon and select `Developer Console` to open a new console window
```

Comment:
Please add a period at the end of each sentence that is missing one in all of the following sections. Please also see the above edits and apply them to the similar sentences below.